src/clj/game/cards/icebreakers.clj

```diff
@@ -767,6 +773,21 @@
                     {:abilities [(break-sub 1 1 "Sentry")
                                  (strength-pump 2 1 :all-run)]})
 
+   "Puffer"
```

Comment:
Could maybe add functions that increase / decrease MU usage of Runner to help with this? And on trash reduce MU-usage by number of power counters?